.github/workflows/cancel-previous-runs.yml

```
@@ -0,0 +1,13 @@
+name: cancel previous runs
+on: [push]
+jobs:
+  cancel:
+    name: 'Cancel Previous Runs'
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - uses: styfle/cancel-workflow-action@0.7.0
```

```suggestion
      - uses: styfle/cancel-workflow-action@514c783
```

This is probably being paranoid, but this protects us from the maintainer swapping the `0.7.0` tag to something malicious that steals our secrets.

---

Ah💡 It makes sense!

---

I have tested the suggestion in my sandbox repo and confirmed it works fine. 😃 

https://github.com/ackintosh/sandbox/blob/master/.github/workflows/cancel.yaml#L15